common/agentinstall/params.go

```diff
@@ -46,16 +43,12 @@ func WithVersion(version string) func(*Params) error {
 		}
 
 		p.version.minor = strings.TrimPrefix(version, prefix)
-		p.version.betaChannel = strings.Contains(version, "-")
-		if p.version.betaChannel {
-			// Update from `7.41.0-rc.7` to `7.41.0~rc.7-1
-			p.version.minor = strings.ReplaceAll(p.version.minor, "-", "~") + "-1"
-		}
+		p.version.betaChannel = strings.Contains(version, "~")
 		return nil
 	}
 }
 
-// WithAgentConfig sets the configuration of the Agent.
+// WithAgentConfig sets the configuration of the Agent. `{{API_KEY}}` can be used as a placeholder for the API key.
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Have we considered using `DD_API_KEY` environment variable ? Though it could be available only for Docker install

Reply:
Nothing prevent from using `DD_API_KEY` as `{{API_KEY}}` is a placeholder